tests/SampleApps/multilanguage/reactdjango/requirements.txt

```diff
@@ -43,7 +43,7 @@ isort==4.3.4
 jdcal==1.4
 jedi==0.13.2
 kombu==4.2.2.post1
-lazy-object-proxy==1.3.1
```

Comment:
Would you mind providing some context as to why we need to unpin the version used for some packages in these sample apps?

Reply:
The python 3.7 integration test was failing in the nightly pipeline because astroid 2.8.0 depends on lazy-object-proxy>=1.4.0. Removing the package version allowed   pip to resolve the dependency conflict and the test passed locally.